### ISSUE TITLE
To Do list for scripts and other close out stuff for dan

### ISSUE THREAD

**Issue by DanielJWood** [energyapps/app_frame#1]
Hey @chaunceyt, @ernieambrose and @atiqwarraich. 

I know you don't want to have another mouth to feed, re: platforms to follow. But since I'm not on Jira and won't be around much longer, I wanted to capture these to-dos in an organized fashion outside of the README. What will follow will concern the following items:

### eGallon
* Restoring functioning diagnostic scripts for eGallon task and Electricity Utilities task through addition of tabulate python package ([ticket for Chauncey](https://github.com/energyapps/egallon-serve/issues/3))
* Transfer Egallon and Utilities diagnostic emails to Ernie and Atiq ([ticket for Chauncey](https://github.com/energyapps/egallon-serve/issues/4))
* Update eGallon updated on a weekly basis ([ticket for Ernie](https://github.com/energyapps/egallon-serve/issues/5))
* Adding CORS to energy.gov/api to allow energyapps.github.io access ([ticket for Chauncey](https://github.com/energyapps/egallon-serve/issues/2))
* Update eGallon to allow feed from energy.gov/api once CORS is enabled ([ticket for Atiq/Ernie](https://github.com/energyapps/egallon-serve/issues/6))

### Social Media Analytics 
* Add existing scripts to github with sensible Readme (This is done, found [here](https://github.com/energyapps/social-scraper))
* Migrating existing scripts from EC2 to Jenkins ([Ticket for Chauncey](https://github.com/energyapps/social-scraper/issues/1))
* Serve CORS enabled data from energy.gov/api  ([ticket for Chauncey](https://github.com/energyapps/egallon-serve/issues/2))
* Updating energyapps.github.io/social projects to reflect energy.gov/api served data ([Ticket for Atiq/Ernie](https://github.com/energyapps/social/issues/1))

**Comment by chaunceyt** [comment]
@DanielJWood Does UTILITIES_ADMIN support comma separated string of email addresses
i.e. UTILITIES_ADMIN="email@domain.tld,email@domain.tld"

**Comment by chaunceyt** [comment]
I just confirmed that it should work reviewed egallon.sh.

**Comment by DanielJWood** [comment]
@chaunceyt I am uncertain if that will work but it appears that you confirmed that it would in fact work. Is this correct?

**Comment by chaunceyt** [comment]
@DanielJWood Yes the way you're using the mail command it should work. I will test to verify.

**Comment by ernieambrose** [comment]
The eGallon portion of this work has been completed.
